_faq/cep-and-its-importance.md

```diff
@@ -0,0 +1,10 @@
+---
+question: "What is a cEP and why it is so important to do analysis pre-gsoc and design as early as possible, to be finalized in the cEP?"
+---
+A **cEP(coala Enhancement Proposal)** is a proposal proposed to the coala community discussing the new major features that the contributor is intended to implement during GSoC.
```

Comment:
`the contributor is intended to implement` could be reworded to `the contributor intends to implement`